server/NucleiDetection/NucleiDetection.py

```diff
@@ -143,12 +156,18 @@ def main(args):
 
         num_tiles = ts.getSingleTile(**it_kwargs)['iterator_range']['position']
 
-        print 'Number of tiles = %d' % num_tiles
+        print('Number of tiles = {}'.format(num_tiles))
 
-        tile_fgnd_frac_list = htk_utils.compute_tile_foreground_fraction(
-            args.inputImageFile, im_fgnd_mask_lres, fgnd_seg_scale,
-            **it_kwargs
-        )
+        if process_whole_image:
+
+            tile_fgnd_frac_list = htk_utils.compute_tile_foreground_fraction(
+                args.inputImageFile, im_fgnd_mask_lres, fgnd_seg_scale,
+                **it_kwargs
+            )
+
+        else:
+
+            tile_fgnd_frac_list = [1.0] * num_tiles
```

Comment:
This should be `tile_fgnd_frac_list = np.full(num_tiles, 1.0)`, because we expect it to be a numpy array when we compute the nonzero count.